packages/flutter/lib/src/foundation/licenses.dart

```diff
@@ -288,6 +288,9 @@ class LicenseEntryWithLineBreaks extends LicenseEntry {
 ///  * [AboutListTile], which is a widget that can be added to a [Drawer]. When
 ///    tapped it calls [showAboutDialog].
 class LicenseRegistry {
+  // This class is intended to be used as an interface, and should not be
```

Comment:
This one is also not meant to be instantiated.

Reply:
Done